always do a bootstrap w/local jimsh0 to avoid requiring tcl or jimtcl
to be installed first.  the bootstrap prog is small too.

--- a/autosetup/find-tclsh
+++ b/autosetup/find-tclsh
@@ -4,9 +4,6 @@
 d=`dirname "$0"`
 { "$d/jimsh0" "$d/test-tclsh"; } 2>/dev/null && exit 0
 PATH="$PATH:$d"; export PATH
-for tclsh in jimsh tclsh tclsh8.5 tclsh8.6; do
-	{ $tclsh "$d/test-tclsh"; } 2>/dev/null && exit 0
-done
 echo 1>&2 "No installed jimsh or tclsh, building local bootstrap jimsh0"
 for cc in ${CC_FOR_BUILD:-cc} gcc; do
 	{ $cc -o "$d/jimsh0" "$d/jimsh0.c"; } 2>/dev/null || continue
